java/libraries/io/src/processing/io/GPIO.java

```diff
@@ -165,7 +165,7 @@ protected static void checkValidPin(int pin) {
    *  the sketch is going to achieve.
    *  @param ms milliseconds to pause
    */ 
-  protected static void delay(int ms) {
+  public static void delay(int ms) {
```

Comment:
Makes sense! Those two functions are gone in later commits to this PR (below) - setBrightness is now brightness as well.